utils/oscap-tool.h

```diff
@@ -37,6 +37,7 @@
 #include <oval_probe.h>
 #include <cvss_score.h>
 #include <xccdf_benchmark.h>
+#include <xccdf_session.h>
```

Comment:
It's odd that you have to include a different file just because of the cmake port, can you explain this? The commit says "Add missing includes" which is far from a good commit message :-/

Reply:
The reason for that was many compiler warnings like:
```
[ 77%] Building C object utils/CMakeFiles/oscap.dir/oscap-tool.c.o
In file included from /home/jcerny/openscap/utils/oscap-tool.c:27:0:
/home/jcerny/openscap/utils/oscap-tool.h:174:47: warning: ‘struct xccdf_session’ declared inside parameter list will not be visible outside of this definition or declaration
 int xccdf_set_profile_or_report_bad_id(struct xccdf_session *session, const char *profile_id, const char *source_file);
```
and similar.

Reply:
Shouldn't that be fixed in maint-1.2 as well then?

Reply:
Yes, that is a great idea, I wasn't aware of them there. I have opened a pull request in https://github.com/OpenSCAP/openscap/pull/901 on maint-1.2.